include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_radix_sort.h

```diff
@@ -297,7 +297,7 @@ __radix_sort_count_submit(_ExecutionPolicy&& __exec, ::std::size_t __segments, :
 
     const auto __val_buf_size = __val_rng.size();
     // iteration space info
-    const auto __blocks_total = __get_roundedup_div(__val_buf_size, __block_size);
```

Comment:
I believe that should be rolled back. But I guess, I now, why you are doing so. It's just a test for your patch